ginga/canvas/types/astro.py

```diff
@@ -656,19 +709,19 @@ def __init__(self, x, y, radius, width=None,
         coord = kwdargs.get('coord', None)
 
         klass = get_canvas_type(atype)
-        obj1 = klass(x, y, radius, color=color,
+        obj1 = klass(pt[0], pt[1], radius, color=color,
```

Comment:
Why not use the new class that takes point here?

Reply:
I think because we are looking up the class based on the string in
```python
klass = get_canvas_type(atype)
```
and these classes are not yet added to the drawing dictionaries.  I wanted to put in the support but not actually modify the drawing (class) dictionaries yet.